### DISCUSSION TITLE
[Feature reqeust] add a third type: tally type

### DISCUSSION THREAD

**Discussion by formula-spectre** [iSoron/uhabits#1874] [Feature Requests]
it would be nice to have an habit that it's more of a "tally", like when you go to input the number, you have +1/-1 at the bottom, but you can still edit the overall number;
my use case is pull ups; since I do them during the day, and I use the home screen widget, I would find it more ergonomic to be able to +1 to the number.  
and yes, I can see the current number, delete it, and add n+1, but I think it would still be a nice feature :)
let me know what you think!

**Comment by RadekPilich** [comment]
I am new user and also miss this the most for ease of logging measurable habits. 

New habit type is not necessary in my opinion though, adding -1/+1 buttons to the value entry pop-up of the current measurable habit would be good enough for me.

Until this happens, I will have to script some Tasker automation, so that +1 is added every time I open the value entry pop-up from a home screen widget.

It has to take one or max two taps to add +1. Three or more taps is not good enough workflow.

**Comment by RadekPilich** [comment]
Or even add -10 -5 -1 +1 +5 +10 buttons.

**Reply by formula-spectre** [reply]
sounds good!

**Comment by RadekPilich** [comment]
Btw. it actually only takes 1(widget) + 2 (new value + ok) taps to log single digit measures, which is pretty much as good as it gets in terms of workflow. The issue the unnecessary cognitive load it takes to increase the value. We have to look at what the current value is, than locate the correct higher value on the keypad, tap it, than tap ok. And this has to occur with each increase, which is mentally taxing and frustrating. Adding the ± buttons eliminates some of these usually unnecessary steps and allows for quick and simple increases that leverage muscle memory.

**Reply by formula-spectre** [reply]
agreed!

**Comment by Wrongkalonka** [comment]
For me, a widget with +1 for a mesurable habit would be a great addition.
I'm using the app to track different things through out the day so most of the time a +1 to the existing number would be all I need.

**Comment by herrbrixter** [comment]
Related to #994
